### ISSUE TITLE
TEI Publisher Integration

### ISSUE THREAD

**Issue by fechan** [eba-diary/Travelogues-webapp#21]
This issue is for tracking important information about the ongoing TEI Publisher integration feature branch.
---
It is possible to access content from another TEI document in the same exist-db instance with a xquery like the following, meaning it is possible to access additional info from something like a TEI site index. 
```xquery
doc("/db/apps/tei-publisher/data/playground/locations.txt.tei")//settlement//text()
```
Of course, if the location of the external document changes within the exist-db instance, this will break.

Basically, wherever you need to display additional info, you put a similar xquery in the corresponding ODD element spec. In the following screenshot, the `alternate` behavior opens a popover on hover of a persName element, with its contents dictated by the parameter named `alternate`.
![image](https://user-images.githubusercontent.com/56131910/103424837-731c1d80-4b63-11eb-9ea0-3d77fbe84776.png)

**Comment by fechan** [comment]
Not a web app thing, but the TEI Publisher server will refuse to accept plaintext documents, meaning you have to convert any plain text documents you want to serve to a TEI file anyway.

**Comment by fechan** [comment]
Other possibly useful xquery statements:
* `string-join((@ref, "Hello"), " ")` takes the `@ref` attribute value of the current node and joins it with the string "Hello"
* `root()//placeName//text()` takes the text of any placeName anywhere in the root of the entire TEI document
